### PR TITLE
Block access rse

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f08d2796f4/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a05584618e/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-61c38e655f/how-to/)

**Comment by stephamon** [comment]
Hey @cyrichardson can you give this an editing pass? Thanks!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-70a95c3279/how-to/)

**Comment by cyrichardson** [comment]
@stephamon Will do!

**Comment by cyrichardson** [comment]
@stephamon Nate specifically asked for a doc file so all the edits would be in one place and not make this page look busy with edits. Do you prefer that as well or would you like inline edits in the changed file in the PR?

**Comment by cyrichardson** [comment]
Here's a doc file with edits: [block-or-restore-rackspace-email-access.docx](https://github.com/rackerlabs/rackspace-how-to/files/1612831/block-or-restore-rackspace-email-access.docx)

Let me know if you have any trouble opening the file or if you have any questions. 

If you'd like the edits inline, just let me know and I will transfer them over.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1f73ccf180/how-to/)

File: content/rackspace-email/block-or-restore-rackspace-email-access.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+**Warning:** Disabling a mailbox does not remove it from your billing. The mailbox continues to receive and store mail even after access is blocked.
+
+### Block mailbox access
+
+1. Log in to the [Cloud Office Control Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=%2f "Cloud Office Control Panel") by using your Rackspace Cloud Office admin ID and password.
+2. In the **Rackspace Email** section, click **Mailboxes**.
+
+   <img src="{% asset_path rackspace-email/block-or-restore-rackspace-email-access/add-mailbox-sc1.png %}" />
+
+3. If you have multiple domains, select the domain for the mailbox to which you intend to block access.
```
Comment (catlook) [inline content/rackspace-email/block-or-restore-rackspace-email-access.md]: ... mailbox for which ...

File: content/rackspace-email/block-or-restore-rackspace-email-access.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+4. Next, you see a list of your mailboxes. In the far-right column, **Action**, click **Block Access** to block access to the corresponding mailbox in that row.
+
+   <img src="{% asset_path rackspace-email/block-or-restore-rackspace-email-access/list_block_access.png %}" />
+
+5. A message displays confirming that you want to block access. Read the message and then click **YES, BLOCK ACCESS**.
```
Comment (catlook) [inline content/rackspace-email/block-or-restore-rackspace-email-access.md]: ... message, and then ...

File: content/rackspace-email/block-or-restore-rackspace-email-access.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+   <img src="{% asset_path rackspace-email/block-or-restore-rackspace-email-access/block_pop_up.png %}" />
+
+The mailbox is then greyed out in the mailbox list. You have now blocked users from accessing that mailbox.
```
Comment (catlook) [inline content/rackspace-email/block-or-restore-rackspace-email-access.md]: ... mailbox displays greyed ...

File: content/rackspace-email/block-or-restore-rackspace-email-access.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+### Restore mailbox access
+
+1. Log in to the [Cloud Office Control Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=%2f "Cloud Office Control Panel") by using your Rackspace Cloud Office admin ID and password.
+2. In the **Rackspace Email** section, click **Mailboxes**.
+
+   <img src="{% asset_path rackspace-email/block-or-restore-rackspace-email-access/add-mailbox-sc1.png %}" />
+
+3. If you have multiple domains, select the domain for the mailbox to which you intend to restore access.
```
Comment (catlook) [inline content/rackspace-email/block-or-restore-rackspace-email-access.md]: ... mailbox for which ...

File: content/rackspace-email/block-or-restore-rackspace-email-access.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+4. You will see a list of your mailboxes. In the far right column **Action**, click **Restore Access** to restore access to the corresponding mailbox in that row.
```
Comment (catlook) [inline content/rackspace-email/block-or-restore-rackspace-email-access.md]: Next, you see a list ....   (get rid of the "will" language)

File: content/rackspace-email/block-or-restore-rackspace-email-access.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+    <img src="{% asset_path rackspace-email/block-or-restore-rackspace-email-access/restore_access.png %}" />
+
+5. A message displays confirming that you want to restore access. Read the message and click **YES, RESTORE ACCESS**.
+
+    <img src="{% asset_path rackspace-email/block-or-restore-rackspace-email-access/restore_pop_up.png %}" />
+
+The mailbox is no longer greyed out in the mailbox list. Users can now access that mailbox.
```
Comment (catlook) [inline content/rackspace-email/block-or-restore-rackspace-email-access.md]: ... mailbox no longer displays greyed out ...

File: content/rackspace-email/block-or-restore-rackspace-email-access.md (hunk 1 above)
Comment (stephamon) [reply content/rackspace-email/block-or-restore-rackspace-email-access.md]: You block access to a mailbox, not for a mailbox

File: content/rackspace-email/block-or-restore-rackspace-email-access.md (hunk 3 above)
Comment (stephamon) [reply content/rackspace-email/block-or-restore-rackspace-email-access.md]: This was a Catherine suggestion

File: content/rackspace-email/block-or-restore-rackspace-email-access.md (hunk 4 above)
Comment (stephamon) [reply content/rackspace-email/block-or-restore-rackspace-email-access.md]: You restore access to a mailbox, not for

File: content/rackspace-email/block-or-restore-rackspace-email-access.md (hunk 6 above)
Comment (stephamon) [reply content/rackspace-email/block-or-restore-rackspace-email-access.md]: This was a Catherine suggestion

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3b4c754151/how-to/). :bow:

**Pull request merged**